webdriver-spec.html

```diff
@@ -3840,8 +3838,20 @@
   as the <a>current browsing context</a>’s <a>active document</a>,
   return an empty sequence.
 
+ <li><p>Let <var>rectangles</var> be
+  the <a><code>DOMRect</code></a> sequence
+  returned by calling <a>getClientRects</a> on <var><a>element</a></var>.
+
+ <!--
+  An element which style property `display` is "none"
+  returns an empty DOMRect sequence.
+ -->
+ <li><p>If <var>rectangles</var> has the length of 0,
+  return an empty sequence.
+
  <li><p>Let <var>centre point</var> be
-  the <a>in-view centre point</a> of <var>element</var>.
+  the <a>in-view centre point</a> of <var>element</var>
+  given the first indexed element of <var>rectangles</var>.
```

Comment:
Maybe we should do something more sophisticated here, but I feel we should land this change and file an issue about the edge case.